test-suite/src/basic.rs

```diff
@@ -64,6 +71,143 @@ CREATE TABLE TestA (
             Ok(select!(id | num; I64 | I64; 2 2; 2 9; 2 4; 2 7)),
             "SELECT id, num FROM Test",
         ),
+        (
+            // SELECT without Table
+            Ok(select!(
+                1   | 'a'       | true | "1 + 2" | "'a' || 'b'"
+                I64 | Str       | Bool | I64     | Str;
+                1     "a".into()  true   3         "ab".into()
+            )),
+            "SELECT 1, 'a', true, 1 + 2, 'a' || 'b'",
+        ),
```

Comment:
`Series` is too big and meaningful to exist only as a part of `basic.rs` tests.
Let's make `test-suite/src/series.rs` to contain these. :D